### PR TITLE
fix(stickyComment): #129 change phrasing to `OP has awarded N deltas`

### PR DESCRIPTION
**Pull request by yleong**
This is for #129, I followed the suggestion of @wyantb and passed the hidden params over to `findOrMakeStickiedComment`

There is no extra API call to retrieve hidden params, but there is 1 extra API call to update the stickied comment.
  
Tested [here](https://www.reddit.com/r/changemyviewDB3Dev2/comments/60hc8y/cmv_i_think_that_bar_should_be_fo/) and [here](https://www.reddit.com/r/changemyviewDB3Dev2/comments/5zyugv/cmv_i_think_that_foo_should_be_bar/)

### CODE REVIEW

**Comment by MystK** [comment]
First of all, thank you for your contribution and thank you for following the commit message! :)

I am hoping others can review the work and provide some comments. I won't be back home till tomorrow and I will bring it in then. :)

Thanks again! Please make sure you add yourself to the contributor's list. ;)

**Comment by Snorrrlax** [comment]
Thanks a lot!

Would it be possible to copy the wiki page code that changes the word "delta" to "deltas" when there's more than 1? Like [here](https://www.reddit.com/r/changemyview/wiki/user/rokmwi) and [here](https://www.reddit.com/r/changemyview/wiki/user/cacheflow). This would be preferable to delta(s).

File: src/index.js [hunk 1]
```diff
@@ -389,7 +389,33 @@ const wasDeltaMadeByAuthor = comment => comment.link_author === getCommentAuthor
 /* Invoked after the DeltaLog post is made, so `deltaLogKnownPosts` will be populated */
 const deltaLogStickyTemplate = _.template(i18n[locale].deltaLogSticky)
 const findOrMakeStickiedComment = async (linkID, comment, deltaLogPost) => {
-  if (deltaLogPost.stickiedCommentID) {
+  const stickyID = deltaLogPost.wikientry.stickiedCommentID
+  const opName = deltaLogPost.postentry.opUsername
+  const deltsAwardedByOP = deltaLogPost.postentry.comments.filter(
```
Comment (RaoulMeyer) [inline src/index.js]: There's a typo in the var name here.

File: src/index.js [hunk 2]
```diff
@@ -389,7 +389,33 @@ const wasDeltaMadeByAuthor = comment => comment.link_author === getCommentAuthor
...
+    comm => comm.awardingUsername === opName
+  ).length
+
+  if (stickyID) {
+    // Update the N in 'OP has awarded N deltas...'
+    const stickyCommentBody = deltaLogStickyTemplate({
+      username: opName,
+      linkToPost: `/r/${deltaLogSubreddit}/comments/${deltaLogPost.wikientry.deltaLogPostID}`,
+      deltaLogSubreddit,
+      opawarded: deltsAwardedByOP,
```
Comment (RaoulMeyer) [inline src/index.js]: I'd suggest adding the request of Snorrrlax here, by adding the word 'delta' or 'deltas' depending on the count of deltasAwardedByOP.

File: src/index.js [hunk 3]
```diff
@@ -389,7 +389,33 @@ const wasDeltaMadeByAuthor = comment => comment.link_author === getCommentAuthor
...
+    })
+
+    const updateParams = {
+      text: `${stickyCommentBody}`,
+      thing_id: `${stickyID}`,
```
Comment (RaoulMeyer) [inline src/index.js]: You don't need the backtick notation here, you can just type:
`text: stickyCommentBody`

File: src/index.js [hunk 4]
```diff
@@ -389,7 +389,33 @@ const wasDeltaMadeByAuthor = comment => comment.link_author === getCommentAuthor
...
+    }
+
+    const updateResponse = await reddit.query({
+      URL: `/api/editusertext?${stringify({ thing_id: `${stickyID}` })}`,
+      method: 'POST',
+      body: stringify(updateParams),
+    })
+    if (updateResponse.error) throw Error(updateResponse.error)
```
Comment (RaoulMeyer) [inline src/index.js]: I don't think you would want to throw this error. Logging it would probably be enough.

**Comment by yleong** [comment]
Thanks for the code review and suggestions. I have implemented and squashed them into 1 commit.

**Pull request merged**